third_party/2and3/sqlalchemy/engine/base.pyi

```diff
@@ -1,20 +1,28 @@
-from typing import Any, List, Tuple
+from typing import Any, Iterable, Iterator, List, Mapping, Optional, Tuple
 
 # Dummy until I figure out something better.
 class Connectable:
```

Comment:
That would be better yes. Not sure how many additional missing methods would have to be added.